tests/integration/valkey-benchmark.tcl

```diff
@@ -12,6 +12,7 @@ proc common_bench_setup {cmd} {
     if {[catch { exec {*}$cmd } error]} {
         set first_line [lindex [split $error "\n"] 0]
         puts [colorstr red "valkey-benchmark non zero code. first line: $first_line"]
+        puts [colorstr red "valkey-benchmark non zero code, the output is: $error"]
```

Comment:
We can keep only the full output. We don't need only the first line.

```suggestion
        puts [colorstr red "valkey-benchmark non zero code, the output is: $error"]
```

Reply:
LGTM, the change will be applied before merging, now waiting for CI to try to finish most of its work.